gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/handlers/api/services/dlq/DlqRequest.java

```diff
@@ -29,4 +30,9 @@ protected DlqRequest(Flowable<Message> messages) {
         super();
         messages(messages);
     }
+
+    @Override
+    public void method(HttpMethod method) {
+        // It is not possible to override DLQ request's method.
+    }
```

Comment:
I'm wondering if we really need to do that because it makes me ask myself "why just method(HttpMethod) and not others" ? As long as it does not affect the DLQ, can we consider that there is no need to override the method ?

Reply:
implementation moved to `AbstractRequest`